pyproject.toml

```diff
@@ -40,6 +40,7 @@ test = [
     "pytest-cov",
     "nbproject-test>=0.4.3",
     "sqlmodel",
```

Comment:
Please remove sqlmodel and the whole dead code in the bionty test! It needs to be re-introduced based on Django - we somehow didn't notice this.